recipes/pytypes/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pytypes" %}
+{% set version = "1.0b3" %}
+{% set sha256 = "20ae0aa741c63126eb697c41ad576239e397436d9fba93934622d0fd7b20dfd6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # url: https://github.com/simplejson/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - typestubs=pytypes.stubfile_2_converter:main
+
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - typing
+
+test:
+  imports:
+    - pytypes
```

Comment:
Would it be possible to put in some more tests? (More import tests are ok)

Reply:
That import basically imports everything in the module, not a very large one